lib/src/supabase.dart

```diff
@@ -39,8 +39,8 @@ class Supabase {
   /// This must be called only once. If called more than once, an
   /// [AssertionError] is thrown
   static Future<Supabase> initialize({
-    String? url,
-    String? anonKey,
+    required String url,
+    required String anonKey,
```

Comment:
Just want to double check, but these are required, right?

Reply:
I think so. At least they are required in the js client: https://supabase.com/docs/reference/javascript/initializing